sbndaq-artdaq/Generators/SBND/TriggerBoardReader_generator.cc

```diff
@@ -26,10 +26,18 @@
 #include <iterator>
 #include <iostream>
 #include <string>
+#include <bitset>
 
 #include <unistd.h>
 #include <thread>
 
+int numGates =0;
```

Comment:
Hey, sorry, I'm again late to the party, but I believe we might need 4 of these counters, similarly to the 4 timestamps? In this case 2 of them would increment on LLT30, two on LLT26 (off-beam), but importantly each resets at the specific HLT type. Otherwise it looks great!